# Setting of map_plane_extraction
MapPlaneExtractionPointInMap: [27.2, -0.2, 2] # set generell part of normals according to convention
                                              # change rest manually in file 

# Settings of CGAL Region Growing for Map plane extraction
CGALRegionGrowMinNumInliers: 130
CGALRegionGrowMaxDistToPlane: 0.01
CGALRegionGrowMaxDistBetwPoint: 7
CGALRegionGrowMaxDiffNormalThreshold: 0.949

CGALRGRegulActive: true
CGALRGRegulParall: true
CGALRGRegulOrthog: true
CGALRGRegulCoplanar: true
CGALRGRegulParallOrthTol: 10
CGALRGRegulCoplanarTol: 0.2


